godtools/Managers/LanguagesManager.swift

```diff
@@ -19,13 +19,29 @@ class LanguagesManager: GTDataManager {
     let path = "/languages"
     
     var languages = [Language]()
+    var selectingPrimaryLanguage = true
     
     override init() {
         super.init()
         serializer.registerResource(LanguageResource.self)
     }
     
-    func loadFromDisk() -> Promise<[Language]> {
+    func loadFromDisk(id: String) -> Language {
+        let language = Language.mr_findFirst(byAttribute: "remoteId", withValue: id)!
```

Comment:
The reason I created this method is to isolate the calls MR calls. Basically I wanted to keep the MR calls for Language in the the LangaugesManager. But I missed one that you previous called out in your other comment.

I want to take another look at context. If we don't specify the context, then it should be the "default" one. In some cases I had to specify it though because I ran into errors about relating objects from two different contexts together. I will take another look and see if that can be standardized at all.